src/modules/endpoint/models/WohnenOfferModel.js

```diff
@@ -24,6 +27,14 @@ class WohnenOfferModel {
   get createdDate (): Moment {
     return this._createdDate
   }
+
+  get formDataType (): Class<F> {
+    return this._formDataType
+  }
```

Comment:
should this really be an attribute of an Event? Seems odd to me...

Reply:
This makes sense so you keep some more typesafety because you can use the flow Class type.
You do not have to use the weird `typeof` feature which is a bit odd to me :/